lib/transform/standardize.js

```diff
@@ -78,7 +78,7 @@ class Standardizer {
     }
 
     if (!this.allowedKeywords.has(keywords[0])) {
-      throw new Error(`Unknown DSL keyword: ${keywords[0]}`);
+      throw new Error(`Unknown Koncorde keyword: ${keywords[0]}`);
```

Comment:
Can we have a custom error with the `keyword` property so we can construct higher level error messages in project using Koncorde?

Reply:
yes that will come soon